test/functional/test_command.py

```diff
@@ -195,6 +200,7 @@ def test_command_dependency_gilt(scenario_to_test, with_scenario,
         ('dependency', 'docker', 'shell'),
         ('dependency', 'ec2', 'shell'),
         ('dependency', 'gce', 'shell'),
+        ('dependency', 'linode', 'gilt'),
```

Comment:
maybe?
```suggestion
        ('dependency', 'linode', 'shell'),
```
did you run this?

Reply:
Yep, needed that. Thanks.